frontend/packages/network-attachment-definition-plugin/src/components/network-attachment-definitions/NetworkAttachmentDefinitionsForm.tsx

```diff
@@ -198,15 +212,32 @@ const NetworkAttachmentDefinitionFormBase = (props) => {
   const [error, setError] = React.useState(null);
   const [fieldErrors, setFieldErrors] = React.useState<FieldErrors>({});
 
-  const networkTypeDropdownItems = getNetworkTypes(hasSriovNetNodePolicyCRD, hasHyperConvergedCRD);
-
   const formIsValid = React.useMemo(
     () => validateForm(fieldErrors, name, networkType, typeParamsData, setError),
     [fieldErrors, name, networkType, typeParamsData],
   );
 
-  React.useEffect(() => setLoading(hasSriovNetNodePolicyCRD && !loaded), [
+  const [networkConfig, networkConfigLoaded] = useK8sWatchResource<K8sResourceKind>({
```

Comment:
If more networks are added, we may want to move this logic into individual hooks to keep the logic separate.